app/models/form_profile.rb

```diff
@@ -126,7 +127,7 @@ def prefill(user)
     @contact_information = initialize_contact_information(user)
     @military_information = initialize_military_information(user)
     mappings = self.class.mappings_for_form(form_id)
-    form_data = generate_prefill(mappings)
+    form_data = generate_prefill(mappings).deep_transform_keys { |k| k.camelize(:lower) }
```

Comment:
@lihanli @jbalboni This change will transform all the prefill keys to camelCase (updated specs as well). Do you think this is the appropriate way to do this?

Reply:
If I can send the x-key-inflection header, then it shouldn't be necessary. I made a comment about `postal_code` mainly because it looked out of place when the way the rest of the fields were named. Let me look into the header thing before you add this particular change.